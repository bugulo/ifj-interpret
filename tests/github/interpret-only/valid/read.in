42
just a string!
true