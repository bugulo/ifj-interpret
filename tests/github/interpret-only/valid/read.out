42just a string!true